frame/support/src/weights.rs

```diff
@@ -460,13 +465,15 @@ impl<Args, WD, CD, PF> ClassifyDispatch<Args> for FunctionOf<WD, CD, PF> where
 }
 
 // `PaysFee` as a raw value
+#[allow(deprecated)]
 impl<Args, WD, CD> PaysFee<Args> for FunctionOf<WD, CD, Pays> {
 	fn pays_fee(&self, _: Args) -> Pays {
 		self.2
 	}
 }
 
 // `PaysFee` as a closure
+#[allow(deprecated)]
```

Comment:
Aren't all these extra allow's are already handled by the one above the struct?

Reply:
Which struct do you mean?

Reply:
`FunctionOf`

Reply:
So why this should be already handled? `FunctionOf` is marked with `#[deprecated]` and not with `#[allow(deprecated]`.

Reply:
Ahh🙈